ffcompare.py

```diff
@@ -208,7 +218,7 @@ def minimizeOpenMM(Topology, System, Positions):
 
     Returns
     -------
-
+    Topology positions
```

Comment:
specify return object name and data type (as well as the description, which you did)